metric/bigtable/src/main/java/com/spotify/heroic/metric/bigtable/BigtableBackend.java

```diff
@@ -421,6 +449,43 @@ public Statistics getStatistics() {
         return async.collect(fetches, FetchData.collect(FETCH));
     }
 
+    private AsyncFuture<FetchData.Result> fetchBatch(
+        final FetchQuotaWatcher watcher, final MetricType type,
+        final List<PreparedQuery> prepared, final BigtableConnection c,
+        final Consumer<MetricCollection> metricsConsumer
+    ) {
+        final BigtableDataClient client = c.dataClient();
+
+        final List<AsyncFuture<FetchData.Result>> fetches = new ArrayList<>(prepared.size());
+
+        for (final PreparedQuery p : prepared) {
+            QueryTrace.NamedWatch fs = QueryTrace.watch(FETCH_SEGMENT);
```

Comment:
Think it could be left as is to use the same API as the other fetchBatch, and change both in a separate commit